### PR TITLE
Fixes in socket dst

### PR DESCRIPTION
**Pull request by ameeshaagrawal**

### CODE REVIEW

File: contracts/socket/SocketDst.sol [hunk 1]
```diff
@@ -164,43 +178,53 @@ abstract contract SocketDst is SocketBase {
...
-            packetId_,
-            proposalCount_,
+            executionDetails_.packetId,
+            executionDetails_.proposalCount,
             remoteSlug,
             packedMessage,
+            packetRoot,
             plugConfig,
-            messageDetails_.decapacitorProof,
+            bytes(""), //messageDetails_.decapacitorProof,
```
Comment (tHeMaskedMan981) [inline contracts/socket/SocketDst.sol:191]: why are we using empty bytes here? as part of socket, shouldnt we keep this generalized?

Comment (arthcp) [reply contracts/socket/SocketDst.sol]: good catch

Comment (arthcp) [reply contracts/socket/SocketDst.sol]: good catch

Comment (ameeshaagrawal) [reply contracts/socket/SocketDst.sol]: thanks, fixed!

**Pull request merged**